web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx

```diff
@@ -312,7 +313,7 @@ const FieldUpdateModal: React.FC<Props> = ({
               name="multiple"
               valuePropName="checked"
               extra={t("Stores a list of values instead of a single value")}>
-              <Checkbox>{t("Support multiple values")}</Checkbox>
+              <Checkbox disabled>{t("Support multiple values")}</Checkbox>
```

Comment:
I thought you were enabling multi value support? This disabled prop makes the checkbox unclickable, no?